incubator/hnc/Makefile

```diff
@@ -61,7 +61,7 @@ KUSTOMIZE ?= ${CURDIR}/hack/kustomize-3.8.1
 CONTROLLER_GEN ?= ${CURDIR}/bin/controller-gen
 
 # Produce CRDs that work back to Kubernetes 1.11 (no version conversion)
-CRD_OPTIONS ?= "crd:trivialVersions=true"
+CRD_OPTIONS ?= "crd:trivialVersions=false"
```

Comment:
Fix comment. Or maybe just delete these two lines completely - what's the default?

Reply:
The default is only top-level schema no per-version schema. Per-version is introduced in k8s 1.13 so the default is to be compatible with older versions. Updated in the comment and changed the code to `CRD_OPTIONS ?= crd` according to https://book.kubebuilder.io/reference/generating-crd.html#multiple-versions